src/sinks/prometheus/exporter.rs

```diff
@@ -333,8 +339,39 @@ impl MetricNormalize for PrometheusExporterMetricNormalizer {
     }
 }
 
+fn authorized(req: &Request<Body>, auth: &Option<Auth>) -> bool {
+    if let Some(auth) = auth {
+        let headers = req.headers();
+        if let Some(auth_header) = headers.get(hyper::header::AUTHORIZATION) {
+            let encoded_credentials = match auth {
+                Auth::Basic { user, password } => HeaderValue::from_str(
+                    format!(
+                        "Basic {}",
+                        base64::encode(format!("{}:{}", user, password.inner()))
```

Comment:
Looks like a missing import of `base64`.

Reply:
Fixed.

Reply:
Hmm, yeah that is odd that you didn't see the same failure locally.